lib/Logger/Logger.cpp

```diff
@@ -85,19 +78,15 @@ void Logger::log(const Level level,
     }
 }
 
-void Logger::logf(const Level level,
-                     const String& file,
-                     const __FlashStringHelper* function,
-                     uint32_t line,
-                     const char* format, ...) {
+void Logger::logf(const Level level, const String& file, const __FlashStringHelper* function, uint32_t line, const char* format, ...) {
     // reimplement printf method so that we can take dynamic list of parameters as printf does
     // (we can't simply pass these on to existing printf methods it seems)
 
     va_list arg;
     va_start(arg, format);
-    char temp[64];  // allocate a temp buffer
+    char temp[64]; // allocate a temp buffer
     char* buffer = temp;
-    size_t len = vsnprintf(temp, sizeof(temp), format, arg);
+    size_t len   = vsnprintf(temp, sizeof(temp), format, arg);
```

Comment:
That's one whitespace too much.

Reply:
It depends. I enabled alignment of blocks of `=`. If this is not wanted I can switch it off again:

```
AlignConsecutiveAssignments: Consecutive
```

Reply:
as desided in the next comment, leave it turned on :)